tests/decode.cpp

```diff
@@ -124,7 +129,11 @@ int main(int argc, char** argv)
 #endif
 
     // drain the output buffer
-    renderOutputFrames(output, true);
+    if (useSpecified) {
+        if(output->renderFrameCount() < frameCount)
+            renderOutputFrames(output, true);
```

Comment:
renderOuputFrames will output more than one frame. so you can't make judgement here.